src/sync/incomingSyncUtils.js

```diff
@@ -353,8 +360,10 @@ export function createOrUpdateRecord(database, settings, recordType, record) {
         status: STATUSES.translate(record.status, EXTERNAL_TO_INTERNAL),
         confirmDate: parseDate(record.confirm_date),
         theirRef: record.their_ref,
+        linkedRequisition,
       };
       const transaction = database.update(recordType, internalRecord);
+      if (linkedRequisition) linkedRequisition.linkedTransaction = transaction;
```

Comment:
Hmm, I will, in case the future requisition will have a persistent link to transaction.. this field is not present in mSupply so can't be synced.. but if updated, we could miss a .save